### PR TITLE
Chore/ptm scaffold

### PR DESCRIPTION
**Pull request by allenjprice**
This PR scaffolds out the flow from one step to another within the PTM process, with rough copy/ui.

### CODE REVIEW

File: src/cljs/apply/db.cljs [hunk 1]
```diff
@@ -30,13 +30,13 @@
...
 (defn bootstrap [account]
   (merge
-   {:lang    :en
-    :account account
-    nav-path nav-items
-    :route   {:page      :home
-              :path      [:home]
-              :params    {}
-              :requester account}}
+   {:lang            :en
```
Comment (jalehman) [inline src/cljs/apply/db.cljs]: The indentation could probably be fixed here with a little `, f l`.

File: src/cljs/apply/sections/community/select.cljs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   :dispatch [:step/advance]})
+
+
+;; views ========================================================================
+
+
+(defmethod content/view step
+  [_]
+  [:div
+   [:div.w-60-l.w-100
```
Comment (jalehman) [inline src/cljs/apply/sections/community/select.cljs]: This looks like a reusable interface component to me. Something like `(layout/header "Which Starcity..." "Browse our communities...")`

File: src/cljs/apply/sections/community/select.cljs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    [:h1 "Which Starcity communities do you want to join?"]
+    [:p "Browse our communities and learn about what makes each special."]]
+   [:div.page-content.w-90-l.w-100
```
Comment (jalehman) [inline src/cljs/apply/sections/community/select.cljs]: See above—the `div.page-content.w-90-l.w-100` should probably be abstracted into a function.

Comment (allenjprice) [reply src/cljs/apply/sections/community/select.cljs]: good points here (and above) - i've filed a task for this as future work.

File: src/cljs/apply/db.cljs [hunk 4]
```diff
@@ -243,16 +118,19 @@
 ;; implementation of this function and have it just return `true`
 (defn has-next-button? [db]
   true
```
Comment (queenieho) [inline src/cljs/apply/db.cljs]: do we still need this `true` here?

Comment (allenjprice) [reply src/cljs/apply/db.cljs]: We don't really need it, since the note above explains how to override this behavior when we want to do that.

**Pull request merged**